### DISCUSSION TITLE
Costing

### DISCUSSION THREAD

**Discussion by PrateekSharma0409** [ant-media/Ant-Media-Server#5482] [Q&A]
Is there a way we can know the total view on stream after it ended . Like total of 100 people watched it when it was live ?. We want to know how much one particular stream cost us.

**Comment by USAMAWIZARD** [comment]
Currently its not possible to get the stream statistics after the streams Has been Ended.
But you can get these stream statistics during the broadcasting state of the stream.

 [this](https://antmedia.io/rest/#/BroadcastRestService/getBroadcastStatistics) [REST API](https://antmedia.io/rest/#/BroadcastRestService/getBroadcastStatistics).

![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/c2954a2b-1fc1-4ce9-bef7-bf673d153db7)

**Reply by Prateekstupa** [reply]
Hi @USAMAWIZARD thanks for the reply we tried to get the duration of the stream (LiveApp/rest/v2/broadcasts/StreamId) but duration is always 0 . we were using the community version tho .

**Comment by rahul7827** [comment]
Hi @PrateekSharma0409 

You can manage this at your application end as well. By utilizing the callback of webRTCAdaptor object and sending this info to your api (assuming you have application DB).

You can also put the condition if same user re-joins the stream the view counter shouldn't increase.
